build.rs

```diff
@@ -1164,6 +1165,9 @@ fn main() {
         .blocklist_function("y1l")
         .blocklist_function("ynl")
         .opaque_type("__mingw_ldbl_type_t")
+        .default_enum_style(bindgen::EnumVariation::Rust {
+            non_exhaustive: false,
```

Comment:
could we also get rid of this? A feature to choose between exhaustive or non exhaustive enums has been added here :) https://github.com/zmwangx/rust-ffmpeg-sys/pull/48

Reply:
It should be good now. The `default_enum_style(bindgen::EnumVariation::Rust` is needed there, because bindgen changed the default in version 0.62